vacasa/connect/connect.py

```diff
@@ -1277,7 +1278,11 @@ def update_contact_finances_payload(self, contact_id, params: dict):
         """
 
         url = f"{self.endpoint}/v1/contacts/{contact_id}/finances"
-        return self._patch(url, json={'data': {'attributes': params}}, headers=self._headers()).json()
+        r = self._patch(url, json={'data': {'attributes': params}}, headers=self._headers(), request_exceptions=False)
+        if r.status_code == HTTPStatus.NO_CONTENT:
+            return None
+        else:
+            return r.json()
```

Comment:
Because `return None` happens implicitly, you could turn these 4 lines into 2 if you wanted.
```
if r.status_code != HTTPStatus.NO_CONTENT:
    return r.json()
```